terms-of-service.md

```diff
@@ -279,15 +283,15 @@ are entitled to submit such User Content and to grant us such license.
 ## 9. Termination
 
 You may stop using our Service, by deleting your account in Settings
-on your personal account on the website. As a subscriber of the paid
-PRO version, you may terminate your subscription at any time, and will
-have access to the Service for the remaining billing (pre-paid)
+on your personal account on the website. As a subscriber of a paid
+plans, you may terminate your subscription at any time, and will
+have access to the Service for the remaining (pre-paid) billing 
 period. The Company reserves the right to terminate the Service and
 the agreement with you with immediate effect upon written notice to
-you. Users of the paid PRO versions may be entitled to refunds and to
-the extent this is described on our [FAQ page](https://support.whereby.com).
-No users are entitled to refunds upon termination due to breach of
-these Terms.
+you. Users of the paid plans may be entitled to refunds and to
+the extent this is described on our [Support Center](https://support.whereby.com). No users are entitled to refunds upon termination due to breach of
+these Terms. What we consider Fair Use is described in our Support Center. We reserve the right to terminate accounts with usage that exceed the [Fair Use Policy](https://whereby.helpscoutdocs.com/article/353-policies). 
```

Comment:
This is also a breakable deep-link. But might be okay for the nice benefit it'll bring.